gpii/node_modules/flowManager/src/UserLogin.js

```diff
@@ -47,12 +49,22 @@
         });
     };
 
+    gpii.request.flowManager.notifyLogin = function (status, userToken, pcpInterface) {
```

Comment:
No, no, no, no, no, no! No hard dependency binding between UserLogin and the pcpInterface. We already have an event representing userLogin - the pcpInterface should just listen to that.